src/main/java/edu/kis/vh/nursery/datastructures/IntArrayStack.java

```diff
@@ -1,22 +1,24 @@
-package edu.kis.vh.nursery;
+package edu.kis.vh.nursery.datastructures;
 
-public class IntArrayStack {
+import edu.kis.vh.nursery.Rhymer;
+
+public class IntArrayStack implements Rhymer {
```

Comment:
Names of operations do not correspond to operating the stack, thus do not reflect the class name.